processing/src/main/java/io/druid/segment/incremental/IncrementalIndex.java

```diff
@@ -114,8 +114,9 @@ public Long apply(final Object o)
         return null;
       }
       if (o instanceof String) {
+        String s = (String) o;
         try {
-          return Long.valueOf((String) o);
+          return s.isEmpty() ? null : Long.valueOf(s);
```

Comment:
I don't think it could, since with the old code, `Long.valueOf("")` would have thrown an exception. The new code would return null.